synapse/federation/transport/client.py

```diff
@@ -995,19 +995,20 @@ async def get_space_summary(
                returned per space
             exclude_rooms: a list of any rooms we can skip
         """
+        # TODO When switching to the stable endpoint, use GET instead of POST.
```

Comment:
we're already using GET? what's left to do?

Reply:
Oh, hmm...I think I should not have included the changes to this file actually, as this will break federation of this feature until other users upgrade.

Do you think it is best to:

* Leave this as is and potentially break federation,
* Leave this as a `POST` for now and update it in the future

Reply:
yeah, I think it does, annoyingly.

Reply:
I backed out the changes from here, minus the comment which now makes sense. 😄